
newer EU::MM versions (notably the version included in perl-5.22.0)
enclose variable references in double-quotes
this patch fixes razor man page installation when such a version is
used, while (supposedly :-) not breaking on older perl installations

--- Makefile.PL.orig	2007-05-09 00:47:53.000000000 +0200
+++ Makefile.PL	2015-06-05 09:41:51.567476568 +0200
@@ -58,9 +58,12 @@
 	   docs/razor-whitelist.pod  \
 	   docs/razor-agents.pod
 INST_MAN5DIR = blib/man5
-INSTALLMAN5DIR = $(DESTDIR)$(PREFIX)/share/man/man5
-INSTALLSITEMAN5DIR = $(DESTDIR)$(SITEPREFIX)/share/man/man5
-INSTALLVENDORMAN5DIR = $(DESTDIR)$(VENDORPREFIX)/share/man/man5
+INSTALLMAN5DIR = $(PREFIX)/share/man/man5
+INSTALLSITEMAN5DIR = $(SITEPREFIX)/share/man/man5
+INSTALLVENDORMAN5DIR = $(VENDORPREFIX)/share/man/man5
+DESTINSTALLMAN5DIR = $(DESTDIR)$(INSTALLMAN35IR)
+DESTINSTALLSITEMAN5DIR = $(DESTDIR)$(INSTALLSITEMAN5DIR)
+DESTINSTALLVENDORMAN5DIR = $(DESTDIR)$(INSTALLVENDORMAN5DIR)
 MAN5EXT = 5
 # end razor-agents
 
@@ -139,10 +142,9 @@
   my $self = shift;
   my $inherited = $self->SUPER::install(@_);
 
-  my $man5 = q{ \\
-		$(INST_MAN5DIR) $(INSTALLMAN5DIR)};
+  my $man5 = q{$(INST_MAN5DIR)};
 
-  $inherited =~ s/(\$\((?:DEST)?INSTALL\w*MAN1DIR\))/$1$man5/gm;
+  $inherited =~ s/((("?)\$\((?:DEST)?INSTALL\w*MA)N1(DIR\)"?))/$1 \\\n\t\t$3$man5$3 $2N5$4/gm;
 
   return $inherited;
 }
